src/Analyzers/CSharp/CodeFixes/ConvertToRecord/ConvertToRecordEngine.cs

```diff
@@ -264,8 +264,12 @@ await RefactorInitializersAsync(type, solutionEditor, propertiesToAssign, cancel
                 }
             }
 
-            var lineFormattingOptions = await document
-                .GetLineFormattingOptionsAsync(fallbackOptions: null, cancellationToken).ConfigureAwait(false);
+#if CODE_STYLE
+            var lineFormattingOptions = LineFormattingOptions.Default;
```

Comment:
Are these VS-only options that are not supported in editorconfig?

Reply:
Not sure.  I'm not really understanding how these options work :D

Reply:
Ok.  found a way to handle this.